active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -118,10 +115,41 @@ thread.Start();
 
 Attempting to activate (`new`) a new Window in a UWP app will fail and log a debug message.
 
-> Implementation note: This will use [RoOriginateError](https://docs.microsoft.com/en-us/windows/win32/api/roerrorapi/nf-roerrorapi-rooriginateerror) to show an explanatory message in the debugger.
+> **Implementation note:** This will use [RoOriginateError](https://docs.microsoft.com/en-us/windows/win32/api/roerrorapi/nf-roerrorapi-rooriginateerror) to show an explanatory message in the debugger.
 
 Creating a new Window in a Desktop app creates a new top level HWND.
 
+### Custom Window's title bar 
+
+SetTitleBar makes a UIElement interact with the system as if it’s the title bar. For example, a user can move the window by dragging the XAML UIElement, or invoke the window context menu by right-clicking it. As a consequence the app no longer receives pointer input when the user interacts with the target UIElement or its children using touch, mouse, or pen. Although the UIElement stils receive keyboard input.
+
+Only one UIElement can act as Title bar, so the last set wins. To use multiple objects, developers need to wrap them in a container element (e.g. a Panel like a Grid)
+
+This method is typically used within the **new** Window's ExtendContentIntoTitleBar property set to true in order to hide the default system title bar. However, even when the default system title bar is not hidden, SetTitleBar can be used to make additional regions in your app behave like the title bar. Extending the content into the title bar do not impact on the Window buttons (Minimize, Maximize, and Close); the buttons will be still there. 
+
+Here there is an example:
+
+```CS
+
+//Asumming MainWindow has a CustomTitleBar public property which is a UserControl, and this UserControl also has some public properties for customization
+private Window m_window;
+
+protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
+{
+    m_window = new MainWindow();
+
+    //Assuming TitleBar never is null
+    m_window.ExtendContentIntoTitleBar = true;
+    m_window.SetTitleBar(m_window.CustomTitleBar);
```

Comment:
Why not just `m_window.TitleBar = m_window.CustomTitleBar;` ?

Reply:
This is good question. I just kept it for compatibility with UWP because it's an existing API, but I don't know the reasons of why we did in this way long time ago. @MikeHillberg any idea?

Reply:
I don't recall either. I do remember we talked about adding a Get method. We in general have a guideline that property setters and getters should be trivial, and if they do any real work should be methods (though we've probably violated that in a few places to be friendlier to markup). Maybe that kicked in here.

Reply:
Leaving this as a method since it's an existing API.